be/app/Http/Controllers/AuthController.php

```diff
@@ -45,4 +46,15 @@ public function register(RegisterRequest $request)
             'token' => $user->createToken('API Token of' . $user->name)->plainTextToken
         ]);
     }
+
+    public function logout()
+    {
+        Session::flush();
+
+        Auth::logout();
+
+        return $this->success([
+            '', 'LogoutSuccessful', 200
+        ]);
+    }
```

Comment:
- do not add newline. You only add newline every end of the function.
- and, use laravel helper in that case you will be no longer need to import.
```suggestion
    public function logout()
    {
        session()->flush();
        auth()->logout();

        return $this->success([
            '', 'LogoutSuccessful', 200
        ]);
    }
```